gcloud-java-bigquery/README.md

```diff
@@ -203,7 +203,9 @@ while (rowIterator.hasNext()) {
 Here we put together all the code shown above into one program. This program assumes that you are
 running on Compute Engine or from your own desktop. To run this example on App Engine, simply move
 the code from the main method to your application's servlet class and change the print statements to
-display on your webpage.
+display on your webpage. Complete source code can be found at
```

Comment:
I vote we keep the reference and remove the embedded code.  It's clear enough and also more maintainable.